style.css

```diff
@@ -26,10 +26,49 @@
   width: 100%;
 }
 
+/* NAV Links */
+
+.hidden {
+  display: none !important;
+}
+
 .nav-menu {
-  display: none;
+  display: flex;
+  position: absolute;
+  width: 100vh;
+  height: 180vh;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #3c3a39;
+  padding: 0 0 0 1em;
+}
+
+.nav-links {
+  margin: 45vh 0 0 0;
+}
+
+.nav-menu a {
+  display: block;
+  color: #fff !important;
+  align-items: center;
+  border-bottom: 1px solid #6f6c6b;
+  font-family: 'Inter', sans-serif;
+  font-size: 2rem;
+  margin: 2rem 0 0 0;
+  padding: 0 0 1rem 0;
+  width: 50%;
 }
 
+.portfolio-nav-x {
+  margin: 0 0 0 80vw;
+}
+
+.portfolio-nav-x-i {
```

Comment:
- [ ] Kindly fix the mobile menu as it's not disappeared when a menu item is clicked. The requirement is it should disappear and the specific part is displayed.

https://user-images.githubusercontent.com/70269507/196661422-c2245358-0413-4525-bfdf-d93e991fccd0.mov

Reply:
@sja-thedude I've made changes as you instructed